test/test_citusdb.yml

```diff
@@ -0,0 +1,256 @@
+---
+- hosts: citus
```

Comment:
This is where I was thinking we should use the variable to determine which host this runs from.  If it is set to `citus`, then the code below will be run on every Citus machine (so 13 times in Production), when we only want it to run once.  I think if you do `hosts: {{ citus_client_host }}`, then we can pass `--extra_vars={'citus_client_host': 'wdcddtl01'}` or whatever to run it from a remote machine.